logic/componentList.js

```diff
@@ -1,16 +1,19 @@
 export const componentsList = [
-    // MUI Components
-    'Button', 'Select', 'Input', 'Grid', 'Card', 'Table', 'TextField', 'Dialog',
+  // MUI Components
+  'Button', 'Select', 'Input', 'Grid', 'Card', 'Table', 'TextField', 'Dialog',
 
-    // Bootstrap Components
-    'Button', 'Dropdown', 'Form', 'InputGroup', 'Card', 'Table',
+  // Bootstrap Components
+  'Button', 'Dropdown', 'Form', 'InputGroup', 'Card', 'Table',
 
-    // Ant Design Components
-    'Button', 'Select', 'Input', 'Grid', 'Card', 'Table', 'Modal',
+  // Ant Design Components
+  'Button', 'Select', 'Input', 'Grid', 'Card', 'Table', 'Modal',
```

Comment:
Let's add eslint and lint pre commit so we have the same formatting please 
Thank you

Reply:
Sure thing